api/v1alpha1/circuitbreaker_types.go

```diff
@@ -46,4 +46,20 @@ type CircuitBreaker struct {
 	// +kubebuilder:validation:Maximum=4294967295
 	// +optional
 	MaxRequestsPerConnection *int64 `json:"maxRequestsPerConnection,omitempty"`
+
+	// Defines per-host Circuit Breaker thresholds
+	// +optional
+	// +notImplementedHide
+	PerHostThresholds *PerHostCircuitBreakers `json:"perHostThresholds,omitempty"`
```

Comment:
Should this be a separate field or something like `maxConnectionsPerHost` since that's the only field currently supported. In the future could add `maxParallelRequestsPerHost`, etc as more support is added.

Reply:
A addtional layer seems more future-proof.